Assembler.UnitTests/AssemblerTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+using Moq;
+using NUnit.Framework;
+
+namespace Assembler.UnitTests
+{
+    [TestFixture]
+    public class MessagesAssemblerTests
```

Comment:
What would happen in `Assemble(BaseFrame)` if `message.FrameType` can't be resolved? (null? exception? etc.)

Reply:
I don't remember what our implementation does, if it returns null or throws an exception.
Do you remember?

Reply:
nope :/ 
guess we'll see once we return

Reply:
Ok